hazelcast-client/src/test/java/com/hazelcast/client/replicatedmap/nearcache/ClientReplicatedMapNearCacheBasicTest.java

```diff
@@ -155,16 +131,4 @@ public void whenRemoveIsUsed_thenNearCacheShouldBeInvalidated_withUpdateOnNearCa
     @Ignore(value = "The ClientReplicatedMapProxy is missing `invalidateNearCache(keyData)` calls")
     public void whenRemoveIsUsed_thenNearCacheShouldBeInvalidated_withUpdateOnDataAdapter() {
     }
-
-    @Test
-    @Override
-    @Ignore(value = "The ClientReplicatedMapProxy is missing `invalidateNearCache(keyData)` calls")
-    public void whenRemoveAsyncIsUsed_thenNearCacheShouldBeInvalidated_withUpdateOnNearCacheAdapter() {
```

Comment:
These kind of methods are actually worse, because they indicate that an async method is broken, but it doesn't exist at all. In this case the sync method is broken, which is internally used by the `DataStructureAdapter`. With this PR this test is just skipped, because the async method doesn't exist in this scenario.